### ISSUE TITLE
 CONNECTUSER not listed in commands

### ISSUE THREAD

**Issue by gajop** [spring/uberserver#155]
What happened to CONNECTUSER? http://springrts.com/dl/LobbyProtocol/ProtocolDescription.html#CONNECTUSER:client
It's not listed in Protocol.py and I'm getting these errors locally:
CONNECTUSER failed. Command does not exist.

Did it get removed completely after https://github.com/spring/LobbyProtocol/issues/19 ? If so, please revert!

**Comment by abma** [comment]
no, it wasn't implemented because of broken/invalid/incomplete protcol spec: https://github.com/spring/LobbyProtocol/issues/19
